### PR TITLE
Sql case issue

### PR DESCRIPTION
**Pull request by lacrimosaprinz**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #

## What this PR does / why we need it:

### CODE REVIEW

**Review by dengn — CHANGES_REQUESTED** [review]
To fully add a bvt case, you need to add also the expected result. In this case, add the mysql result. wrap up the wrong SQL line with the @bvt label.

File: test/cases/subquery/comparisons-using-subqueries.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+insert into t4 values (4,8),(3,8),(5,9);
+insert into t3 values (6),(7),(3);
+-- @bvt:issue#3312
+select * from t3 where a = (select b from t2);
+-- @bvt:issue
+select * from t3 where a = (select distinct b from t2);
+
+
+CREATE TABLE t1 ( a char(1), b char(1));
+INSERT INTO t1 VALUES ('1','1'),('2','2'),('3','3');Query OK, 0 rows affected (0.02 sec)
```
Comment (fengttt) [inline test/cases/subquery/comparisons-using-subqueries.sql:16]: Query OK?

**Review by fengttt — COMMENTED** [review]
Where is the expected result file?   I think this will break bvt.

**Comment by fengttt** [comment]
Reject this issue, as comment has not been addressed for a while and I assume it is abandoned.

**Pull request closed without merging**